pkgs/development/tools/rtags/default.nix

```diff
@@ -1,14 +1,20 @@
-{ stdenv, fetchgit, cmake, llvmPackages, openssl, writeScript, bash, emacs }:
+{ stdenv, lib, fetchgit, cmake, llvmPackages, openssl, writeScript, apple_sdk, bash, emacs }:
 
 stdenv.mkDerivation rec {
   name = "rtags-${version}";
   version = "2.3";
 
-  buildInputs = [ cmake llvmPackages.llvm openssl llvmPackages.clang emacs ];
+  buildInputs = [ cmake llvmPackages.llvm openssl llvmPackages.clang emacs ]
+    ++ lib.optional stdenv.isDarwin apple_sdk.sdk;
 
   preConfigure = ''
-    export LIBCLANG_CXXFLAGS="-isystem ${llvmPackages.clang.cc}/include $(llvm-config --cxxflags)" \
-           LIBCLANG_LIBDIR="${llvmPackages.clang.cc}/lib"
+    export LIBCLANG_CXXFLAGS="-isystem ${llvmPackages.clang.cc}/include $(llvm-config --cxxflags) " \
+
```

Comment:
@LnL7 Good catch. GH hunk preview above was not so informative. Let me push a fix